code/modules/projectiles/gun.dm

```diff
@@ -333,7 +363,12 @@
 	accuracy = initial(accuracy)	//Reset the gun's accuracy
 
 	if(muzzle_flash)
-		set_light(0)
+		//VOREStation Edit - Flashlights
+		if(gun_light)

```

Comment:
I have concerns with this. The last time I added light to guns, it caused immense problems of lag. Keep in mind I had done it so all _projectiles_ would be lit up. This should be a lot less awful but if a major gunfight breaks out it could be really problematic.

Just don't add light to moving projectiles. Lasers are probably fine because they're instant because they're beams. It's the act of lots of bullets traveling across the screen with light on them that horribly bottlenecks the server, causing lag worse than atmosia could imagine.